service.subtitles.legendastv/resources/language/English/strings.po

```diff
@@ -0,0 +1,74 @@
+# Kodi Media Center language file
```

Comment:
Language folder layout is wrong, please use resources/language/resource.language.en_gb/strings.po
Same for the portuguese (pt_pt)